### DISCUSSION TITLE
Font in console

### DISCUSSION THREAD

**Discussion by Aya-Badawy89** [gama-platform/gama#90] [Q&A] [answered]
Hello,

This is a very simple side question, but I cannot resolve it myself
I have noticed that the default size font of the console has decreased so much. I don't know if I clicked on something which made it like that, but it is not like before. Everytime I start any simulation, I need to click on  (increase the font size) from the icon in the console itself. The default was fine before. 
I was wondering if there is something from which we can change the default size font in the console?

**Comment by hqnghi88** [comment, marked as answer]
How about font size in editor (gaml file). Can you try (or did you ever) change this in preference:
<img width="655" alt="image" src="https://user-images.githubusercontent.com/6105384/150649094-d14bc025-a8a8-4f7a-8eb0-bc347bb94f62.png">

**Comment by Aya-Badawy89** [comment]
Oh yes. That did the trick. Thank you very much.
